src/components/modules/blockEvents.ts

```diff
@@ -226,10 +231,22 @@ export default class BlockEvents extends Module {
    * @param {KeyboardEvent} event - keydown
    */
   private backspace(event: KeyboardEvent): void {
-    const BM = this.Editor.BlockManager;
-    const currentBlock = this.Editor.BlockManager.currentBlock,
+    const { BlockManager, BlockSelection, Caret } = this.Editor;
+    const currentBlock = BlockManager.currentBlock,
       tool = this.Editor.Tools.available[currentBlock.name];
 
+    if (currentBlock.selected || BlockManager.currentBlock.isEmpty) {
```

Comment:
надо вынести в переменную или описать этот блок в комметарии
```suggestion
    /**
    * Check if Block should be removed by current Backspace keydown
    */
    if (currentBlock.selected || BlockManager.currentBlock.isEmpty) {
```